From: Pino Toscano <pino@debian.org>
Date: Fri, 22 Jun 2012 14:03:06 +0200
Subject: FTBFS on hurd-i386

Bug-Debian: http://bugs.debian.org/676756

AF_LINK is defined, there is no sockaddr_dl
struct for it.
The attached patch makes netifaces.c pretend there is no AF_LINK
(while still having it defined). I did a quick test, and the built
python-netifaces seems working fine.
---
 netifaces.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/netifaces.c b/netifaces.c
index fa18add..1d50fb5 100644
--- a/netifaces.c
+++ b/netifaces.c
@@ -239,6 +239,8 @@ static int af_to_len(int af)
 #    define AF_LINK  -1000
 #  endif
 #  define HAVE_AF_LINK 0
+#elif defined(__GNU__)
+#  define HAVE_AF_LINK 0
 #else
 #  define HAVE_AF_LINK 1
 #endif
